_components/range-slider/range-slider.md

```diff
@@ -14,6 +14,14 @@ type: component
 maturity: alpha
 id_custom: range
 lead: A range slider allows users to choose an approximate number from a range.
+implementation:
+  initProps:
+    - property: "`data-text-unit`"
+      description: "Adds a unit to screen reader callouts. The unit will be read after the current value."
+      default: "`null`"
+    - property: "`data-text-preposition`"
+      description: "Changes the preposition between current value and max value. If no value is present, it will default to of. When providing a range slider on pages in a language other than English, be sure to update as necessary."
```

Comment:
Had a few suggestions: 
```suggestion
      description: 'Changes the preposition between current value and max value in screen reader readouts. For example, adding a value with the Spanish translation "de" enables a readout like "20 de 100". When adding a range slider to a non-English page, be sure to update the preposition as necessary.'
```

Reply:
Resolved in [16a0ea7](https://github.com/uswds/uswds-site/pull/2310/commits/16a0ea7739e49bfa25480dc73bca6931890624d9)